Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
 
+COPY --from=quay.io/eerez/python-builder:ubi-minimal8.6-3.11.0 /python /python-installation
+
 RUN --mount=type=tmpfs,destination=/var/cache\
     --mount=type=cache,target=/var/cache/yum\
     --mount=type=tmpfs,destination=/root/.cache\
     --mount=type=cache,target=/root/.cache\
-    microdnf update -y && microdnf install -y python3 python3-pip && python3 -m pip install --upgrade pip
-WORKDIR assisted_event_scrape/
+    microdnf update -y && microdnf install make -y && \
+    cd /python-installation && make install && microdnf -y install python3-devel && dnf clean all && python3 -m pip install --upgrade pip
```

Comment:
Please remove `dnf clean` instruction: as we use buildkit in this project, and we have a cache mount on `/var/cache/yum`, if we clean the cache it will not be taken advantage of in later builds. To remove it from burning it from the image, we use `tmpfs` mount type.

Is `/python-installation` directory needed after install?